tango/common/from_params.py

```diff
@@ -421,8 +421,6 @@ def construct_arg(
         if origin is None and isinstance(popped_params, annotation):
             return popped_params
         elif popped_params is not None:
-            subextras = create_extras(annotation, extras)
```

Comment:
Can we get rid of `subextras` everywhere then?

Reply:
Done